### PR TITLE
Fix ot not operator. Search for it only if it is followed by '.'

### PR DESCRIPTION
**Pull request by kouylekov-usit**

### CODE REVIEW

**Comment by leondutoit** [comment]
Thanks! Add a test?

File: pysquril/parser.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def parse_elements(self) -> list:
                 combinator = c.replace(':', '')
                 element = element.replace(c, '')
         term, op_and_val = element.split('=')
-        if 'not' in op_and_val:
+        if 'not.' in op_and_val:
```
Comment (leondutoit) [inline pysquril/parser.py:170]: perhaps it is even more robust to check `if op_and_val.startswith("not.")`

Comment (kouylekov-usit) [reply pysquril/parser.py:170]: Valid

**Comment by leondutoit** [comment]
I'll close this since you made a new one

**Pull request closed without merging**